openapi/openapi.yaml

```diff
@@ -247,3 +247,85 @@ paths:
           description: The current User
         'default':
           description: Error
+components:
+  schemas:
+
+    PageCount:
```

Comment:
Is there a reason of extracting `PageCount` as a standalone schema if it's used only in `PageMeta` schema and it's short. I think it creates too many utility standalone schemas that are not reused anywhere.

Reply:
Yeah, it's def can be inlined.

Reply:
inlined